src/EditorFeatures/CSharpTest/AddUsing/AddUsingTestsWithAddImportDiagnosticProvider.cs

```diff
@@ -28,83 +28,6 @@ public AddUsingTestsWithAddImportDiagnosticProvider(ITestOutputHelper logger)
         internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
             => (new CSharpUnboundIdentifiersDiagnosticAnalyzer(), new CSharpAddImportCodeFixProvider());
 
-        [WorkItem(1239, @"https://github.com/dotnet/roslyn/issues/1239")]
-        [Fact]
-        public async Task TestIncompleteLambda1()
-        {
-            await TestInRegularAndScriptAsync(
-@"using System.Linq;
-
-class C
-{
-    C()
-    {
-        """".Select(() => {
-        new [|Byte|]",
-@"using System;
-using System.Linq;
-
-class C
-{
-    C()
-    {
-        """".Select(() => {
-        new Byte");
-        }
-
-        [WorkItem(1239, @"https://github.com/dotnet/roslyn/issues/1239")]
-        [Fact]
-        public async Task TestIncompleteLambda2()
-        {
-            await TestInRegularAndScriptAsync(
-@"using System.Linq;
-
-class C
-{
-    C()
-    {
-        """".Select(() => {
-            new [|Byte|]() }",
-@"using System;
-using System.Linq;
-
-class C
-{
-    C()
-    {
-        """".Select(() => {
-            new Byte() }");
-        }
-
-        [WorkItem(860648, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/860648")]
-        [WorkItem(902014, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/902014")]
-        [Fact]
-        public async Task TestIncompleteSimpleLambdaExpression()
-        {
-            await TestInRegularAndScriptAsync(
-@"using System.Linq;
-
-class Program
-{
-    static void Main(string[] args)
-    {
-        args[0].Any(x => [|IBindCtx|]
-        string a;
-    }
-}",
-@"using System.Linq;
-using System.Runtime.InteropServices.ComTypes;
-
-class Program
-{
-    static void Main(string[] args)
-    {
-        args[0].Any(x => IBindCtx
-        string a;
-    }
-}");
-        }
-
         [WorkItem(829970, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/829970")]
         [Fact]
         public async Task TestUnknownIdentifierGenericName()
```

Comment:
we still need these tests for incomplete members until compiler fixes: https://github.com/dotnet/roslyn/issues/7536